drivers/misc/cldatadrv/r_taurus_cluster_protocol.h

```diff
@@ -0,0 +1,20 @@
+#ifndef R_TAURUS_CLUSTER_PROTOCOL_H
```

Comment:
This file should go to the `include/uapi/linux` so userspace apps can include it as well. Check out other files in this directory as an example.